lib/metanorma/collection_manifest/collection_manifest.rb

```diff
@@ -0,0 +1,231 @@
+# frozen_string_literal: true
+
+require_relative "../util/util"
+
+module Metanorma
+  # Metanorma collection's manifest
+  class CollectionManifest
```

Comment:
This file should exist at `lib/metanorma/collection_manifest.rb`. Period. This is the Ruby class convention. It existed since Ruby existed.

Reply:
Then I'm going to move all these to be Metanorma::Collection::Manifest